backend/controllers/attendance/attendanceStatistics.js

```diff
@@ -22,19 +22,21 @@ const formatReason = ({ name, triggersIEPWarning }) =>
 
 const buildStatsViewModel = (dashboardStats, triggersIEPWarning, changes) => {
   const mapReasons = reasons =>
-    Object.keys(reasons).map(name => ({
-      id: capitalizeStart(name),
-      name: formatReason({ name, triggersIEPWarning }),
-      value: Number(reasons[name]),
-    }))
+    !reasons
+      ? []
+      : Object.keys(reasons).map(name => ({
```

Comment:
meh.. can get rid of the conditional: `reasons => Object.keys(reasons || {}).map...`